doc/cli/vast-explore.md

```diff
@@ -109,6 +109,22 @@ in the query result.
 
 **Output**
 
+`--max-events=N`:
+
+An invocation of `vast explore` will print at most N events. A value of 0
+means unlimited. This option only restricts the output of `vast explore`,
+internally more than N results may be processed while preparing the results.
+
+`--limit1=NUMBER`, `--limit2=NUMBER`:
```

Comment:
If I understand your choice of numeric suffixes correctly, you chose them because they express the sequencing of the limits. Maybe we can use something that's focusing less on the way the exploration works, but instead how to control the result. My suggestion would be `query-limit` and `context-limit`. I'm not sold on those, but just wanted to start the discussion.